server/sonar-server/src/main/java/org/sonar/server/measure/ws/ComponentAction.java

```diff
@@ -137,6 +153,16 @@ private ComponentWsResponse doHandle(ComponentWsRequest request) {
     }
   }
 
+  private ComponentDto loadComponent(DbSession dbSession, ComponentWsRequest request) {
+    String componentKey = request.getComponent();
+    String componentId = request.getComponentId();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getComponentId"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)